tests/components/lcn/test_services.py

```diff
@@ -0,0 +1,399 @@
+"""Test for the LCN services."""
+
+from unittest.mock import patch
+
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.lcn import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .conftest import MockModuleConnection, MockPchkConnectionManager, setup_component
+
+
+@patch("pypck.connection.PchkConnectionManager", MockPchkConnectionManager)
+async def test_service_output_abs(
+    hass: HomeAssistant, snapshot: SnapshotAssertion
+) -> None:
+    """Test output_abs service."""
+    await async_setup_component(hass, "persistent_notification", {})
+    await setup_component(hass)
+
+    with patch.object(MockModuleConnection, "dim_output") as dim_output:
+        await hass.services.async_call(
+            DOMAIN,
+            "output_abs",
+            {
+                "address": "pchk.s0.m7",
+                "output": "output1",
+                "brightness": 100,
+                "transition": 5,
+            },
+            blocking=True,
+        )
+
+    assert dim_output.await_args.args == snapshot(name="dim_output")
+
+
+@patch("pypck.connection.PchkConnectionManager", MockPchkConnectionManager)
+async def test_service_output_rel(
+    hass: HomeAssistant, snapshot: SnapshotAssertion
+) -> None:
+    """Test output_rel service."""
+    await async_setup_component(hass, "persistent_notification", {})
+    await setup_component(hass)
+
+    with patch.object(MockModuleConnection, "rel_output") as rel_output:
+        await hass.services.async_call(
+            DOMAIN,
+            "output_rel",
+            {
+                "address": "pchk.s0.m7",
+                "output": "output1",
+                "brightness": 25,
+            },
+            blocking=True,
+        )
+
+    assert rel_output.await_args.args == snapshot(name="rel_output")
+
+
+@patch("pypck.connection.PchkConnectionManager", MockPchkConnectionManager)
+async def test_service_output_toggle(
+    hass: HomeAssistant, snapshot: SnapshotAssertion
+) -> None:
+    """Test output_toggle service."""
+    await async_setup_component(hass, "persistent_notification", {})
+    await setup_component(hass)
+
+    with patch.object(MockModuleConnection, "toggle_output") as toggle_output:
+        await hass.services.async_call(
+            DOMAIN,
+            "output_toggle",
+            {
+                "address": "pchk.s0.m7",
+                "output": "output1",
+                "transition": 5,
+            },
+            blocking=True,
+        )
+
+    assert toggle_output.await_args.args == snapshot(name="toggle_output")
```

Comment:
I personally think snapshots work better for service results rather than arguments, but I'll leave that up to you